fixtures/python/Pipfile

```diff
@@ -4,12 +4,14 @@ verify_ssl = true
 name = "pypi"
 
 [requires]
-python_full_version = "3.8.6"
+python_full_version = "3.11.4"
```

Comment:
Would it be possible to keep the Python version more generic to avoid republishing tests if the base Noosenergy CircleCI image changes? The orb executor and the CircleCI image has been kept distinct to keep some agility on upgrading the CI/CD image dependencies.